app/src/main/java/com/example/mymovieinfo/MainActivity.kt

```diff
@@ -22,13 +22,12 @@ class MainActivity : AppCompatActivity() {
         var result = findViewById(R.id.txt_Results) as TextView
 
 
-        val btn_FindMyMovie = findViewById(R.id.btn_FindMyMovie) as Button
-        btn_FindMyMovie.setOnClickListener {
+        val btnFindMyMovie = findViewById(R.id.btn_FindMyMovie) as Button
+        btnFindMyMovie.setOnClickListener {
                 // your code to perform when the user clicks on the button
```

Comment:
Removed underscore from variable name based on Kotlin naming conventions (https://kotlinlang.org/docs/coding-conventions.html#naming-rules)

Reply:
I don't see this change...
We use underscores for constants.  Otherwise, correct, we typically don't use them in variable names.